src/frontend/src/common/components/library/data_subview/index.tsx

```diff
@@ -141,11 +142,22 @@ const DataSubview: FunctionComponent<Props> = ({
   const [isMetadataSelected, setMetadataSelected] = useState<boolean>(false);
   const [isFastaSelected, setFastaSelected] = useState<boolean>(false);
   const [isFastaDisabled, setFastaDisabled] = useState<boolean>(false);
+  const [createTreeModalOpen, setCreateTreeModalOpen] =
```

Comment:
idk if this is much of a pattern at czi, but one nice thing you can do for booleans is to prefix the variable name with something that indicates the value would be true/false. For example `didCreateTreeFailed` vs `createTreeFailed`. You can also see an example of this a few lines above with `isFastaDisabled`. Good prefixes for booleans (not all inclusive!) are `is`, `did`, `has`, `can`. etc. Basically if it's a word you can use to start a yes/no question in English 😂 .

Would love to see that pattern used here as well.

Reply:
yeah i can change them, i think that makes sense. I named the other boolean values in that way, interesting that i didn't do the same here..  lol